tests/shell/Makefile

```diff
@@ -6,10 +6,17 @@ USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
 
+# for z1, socat doesn't work (unknown reason)
+ifeq (z1, $(BOARD))
+  RIOT_TERMINAL ?= pyterm
+endif
+
 # Use a terminal that does not introduce extra characters into the stream.
 RIOT_TERMINAL ?= socat
 
-DISABLE_MODULE += test_utils_interactive_sync
+# the test script skips tests if socat is not used
+include $(RIOTBASE)/makefiles/utils/variables.mk
```

Comment:
Why is this needed it is included in, maybe move after `Makefile.include`?

https://github.com/RIOT-OS/RIOT/blob/06dca07b0a03dd034f85719a1cf7a5266d0aac86/Makefile.include#L143-L145

Reply:
> maybe move after `include $(RIOTBASE)/Makefile.include`?
> 
> [](https://github.com/RIOT-OS/RIOT/blob/06dca07b0a03dd034f85719a1cf7a5266d0aac86/Makefile.include#L143-L145)

that does the trick, fixed